packages/tupaia-web/src/views/RequestProjectAccessModal/RequestProjectAccessModal.tsx

```diff
@@ -84,28 +91,71 @@ export const RequestProjectAccessModal = () => {
   // Show the requested countries if there are any, and the user has not opted to request additional countries
   const showRequestedCountries = requestedCountries?.length > 0 && !requestAdditionalCountries;
 
+  const getOnCloseURI = () => {
```

Comment:
Noticed a bit of a strange UX issue which I think would be related to this logic.

Steps:
-  Navigate to a country without access
   - Request access Modal pops up
- Click 'View other projects'
   - Taken to Select Project modal
- Try to close or exit the modal
   - The user cannot exit the modal

And there's an error that shows in the console log:
```
Uncaught TypeError: searchParamsToRemove.forEach is not a function
    at removeUrlSearchParams (removeUrlSearchParams.ts:9:26)
    at closeModal (useModal.ts:33:15)
    at HTMLUnknownElement.callCallback2 (react-dom.development.js:188:14)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:237:16)
    at invokeGuardedCallback (react-dom.development.js:292:31)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:306:25)
    at executeDispatch (react-dom.development.js:389:3)
    at executeDispatchesInOrder (react-dom.development.js:414:5)
    at executeDispatchesAndRelease (react-dom.development.js:3278:5)
    at executeDispatchesAndReleaseTopLevel (react-dom.development.js:3287:10)
```